storage/linux/storage_if_linux.c

```diff
@@ -239,7 +250,7 @@ int32_t fdo_blob_read(const char *name, fdo_sdk_blob_flags flags, uint8_t *buf,
 			 PLATFORM_HMAC_SIZE, &strcmp_result);
 		if (strcmp_result != 0) {
 			LOG(LOG_ERROR, "%s: HMACs do not compare!\n", __func__);
-			goto exit;
+//			goto exit;
```

Comment:
Why is this commented ? If that is intentional, why are we proceeding as normal flow when HMAC not matching ?